grails-app/views/layouts/main_component.gsp

```diff
@@ -134,6 +134,8 @@
         serverURL: "${webRequest.baseUrl}",
         userNameSearchUrl: "${createLink(controller: 'search', action: 'getMatchingUsers')}",
         sourceDiseases: "${createLink(controller: 'search', action: 'getMatchingDiseaseOntologies')}",
+        unConsentedSampleCollections: "${createLink(controller: 'consentGroup', action: 'unConsentedSampleCollections')}",
+        getConsentGroupSampleCollections: "${createLink(controller: 'consentGroup', action: 'getConsentGroupSampleCollections')}",
```

Comment:
Nit: naming. For lines 136-139, we've broken with the naming convention of having URL be part of the variable name. `unConsentedSampleCollections` sounds like an actual list of unconsented sample collections rather than the url that provides the list of unconsented sample collections. And while `getConsentGroupSampleCollections` is more clear, it also breaks with the pattern of ending with `Url`, as does `sourceDiseases` which is not technically part of this PR. The naming consistency is important so code that uses it is easier to understand. For example, later on in this PR, there is a property assignment: 
```
unConsentedSampleCollections: this.props.unConsentedSampleCollections
``` 
that suspiciously looks like a list of things rather than an API call to get the list of things.

Reply:
I'm seeing this more now that I look ... see `getConsentGroups`, etc. We should try to be more consistent as this gets confusing very fast.